astroid/tests/unittest_inference.py

```diff
@@ -4273,17 +4273,5 @@ def test_duplicated_keyword_arguments(self):
         self.assertIn('f', site.duplicated_keywords)
 
 
-class ObjectDunderNewTest(unittest.TestCase):
```

Comment:
Rather than removing it entirely, this test should be marked as expected failure.

Reply:
@PCManticore Thanks, updated.